src/components/PoolInfoCard.tsx

```diff
@@ -172,6 +174,11 @@ function PoolInfoCard({ data }: Props): ReactElement | null {
           </Typography>
         </InfoItem>
 
+        <Typography>{formattedData.futureA}</Typography>
+        <Typography>
+          {formattedData.futureATime.toLocaleDateString()}
```

Comment:
why is this addition necessary to test wallet?

Reply:
My mistake.
This file is not for this PR.
I think it was added while merge with master. will remove.

Reply:
Removed!